arcgis-java-toolkit-examples/src/main/resources/fxml/app.fxml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import com.esri.arcgisruntime.toolkit.controller.ExampleView?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ScrollPane?>
+<?import javafx.scene.layout.*?>
+<!-- Defines the main layout for the Examples App -->
+<BorderPane fx:id="borderPane" xmlns:fx="http://javafx.com/fxml" fx:controller="com.esri.arcgisruntime.toolkit.controller.ExamplesAppController">
+    <!-- Top section of the BorderPane is the Header bar with menu for selecting an Example -->
+    <top>
+        <HBox fx:id="headerBar" spacing="50" alignment="CENTER" styleClass="panel, panel-large-padding, panel-no-border">
+            <children>
+                <ComboBox fx:id="menu" promptText="Select a tool" id="menu" />
+            </children>
+        </HBox>
+    </top>
+    <center>
+        <!-- Center section of the BorderPane is a StackPane that displays a Landing Page or Example View  -->
+        <StackPane fx:id="mainContent">
+            <children>
+                <!-- Landing Page -->
+                <ScrollPane fx:id="landingPageScrollPane" fitToWidth="true" fitToHeight="true" styleClass="scroll-pane-no-background">
```

Comment:
For cases where we don't need to refer to the node in code, I don't think we need to specify the `fx:id`.